OpenRA.Game/Network/ReplayRecorder.cs

```diff
@@ -69,6 +69,9 @@ void StartSavingReplay(byte[] initialContent)
 
 		public void Receive(int clientID, byte[] data)
 		{
+			if (disposed)
+				return;
```

Comment:
Is this needed to prevent a crash? I feel we should fix the caller to not call methods on a disposed object if they're not going to work any more, rather than hiding the problem here.